netty/src/test/java/io/grpc/netty/NettyClientTransportTest.java

```diff
@@ -344,6 +345,24 @@ public void clientStreamGetsAttributes() throws Exception {
     assertEquals(address, rpc.stream.getAttributes().get(Grpc.TRANSPORT_ATTR_REMOTE_ADDR));
   }
 
+  @Test
+  public void keepAliveEnabled() throws Exception {
```

Comment:
Could you do an RPC in each of these tests and just make sure you get OK? That will check many of the `== null` checks in the code.